### PR TITLE
Add website of awesome-mcp-servers

### PR DESCRIPTION
**Pull request by wong2**
My `awesome-mcp-servers` repo now has a [website](https://mcpservers.org/) for easier viewing and filtering.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ A growing set of community-developed and maintained servers demonstrates various
 Additional resources on MCP.
 
 - **[Awesome MCP Servers by punkpeye](https://github.com/punkpeye/awesome-mcp-servers)** - A curated list of MCP servers by **[Frank Fiegel](https://github.com/punkpeye)**
-- **[Awesome MCP Servers by wong2](https://github.com/wong2/awesome-mcp-servers)** - A curated list of MCP servers by **[wong2](https://github.com/wong2)**
+- **[Awesome MCP Servers by wong2](https://github.com/wong2/awesome-mcp-servers)** - **[website](https://mcpservers.org)** - A curated list of MCP servers by **[wong2](https://github.com/wong2)**
```
Comment (jspahrsummers) [inline README.md:74]:
```suggestion
- **[Awesome MCP Servers by wong2](https://github.com/wong2/awesome-mcp-servers)** (**[website](https://mcpservers.org)**) - A curated list of MCP servers by **[wong2](https://github.com/wong2)**
```

**Review by jspahrsummers — CHANGES_REQUESTED** [review]
Thank you! Just a minor nit for formatting, but this sounds great to include.

**Comment by wong2** [comment]
Thanks! Updated

**Review by jspahrsummers — APPROVED** [review]
Thanks!

**Pull request merged**